react/week3/to-do-app/src/AddToDo.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function AddToDo({addNewTodo, todosLength}){ 
+    const [validDesc, setValidDesc] = useState(true);
+    const [validDate, setValidDate] = useState(true);
+    const [startDate, setStartDate] = useState(new Date());
+    const inFuture = (date) => {
+        const nowDate = new Date();
+        nowDate.setHours(0,0,0,0);
+        date.setHours(0,0,0,0);
+        return date.getTime()>=nowDate.getTime();
+      };
+      const onAddToDo = () => {
+        const newDescription=document.getElementById('newDescription').value;
+        const newDeadLine=startDate.getFullYear()+'-'+(startDate.getMonth()+1)+'-'+startDate.getDate();
+        const validDesc=(newDescription.length>0)?true:false;
```

Comment:
```suggestion
        const validDesc = newDescription.length > 0;
```
In this case you don't need a ternary operator, you can just check if the condition is true or false like this.